stdlib/uuid.ksy

```diff
@@ -0,0 +1,49 @@
+meta:
+  id: uuid
+  endian: le
+  license: Unlicense
+doc: "UUID according to https://www.ietf.org/rfc/rfc4122.txt"
+seq:
+  - id: time_low
+    type: u4be
+    doc: The low field of the timestamp
+  - id: time_mid
+    type: u2be
+    doc: The middle field of thetimestamp
+  - id: time_hi_and_version
+    type: u2be
+    doc: The high field of the timestamp multiplexed with the version number
+  - id: clock_seq_hi_and_reserved
+    type: u1
+    doc: The high field of the clock sequence multiplexed with the variant
+  - id: clock_seq_low
+    type: u1
+    doc: The low field of the clock sequence
+  - id: node
+    type: u4be
+    doc: The spatially unique node identifier
+  - id: node1
+    type: u2be
+    doc: The spatially unique node identifier
```

Comment:
This particular sequence describes only one possible variant, which is the one described in RFC. [Microsoft's version](https://msdn.microsoft.com/en-us/library/windows/desktop/aa373931(v=vs.85).aspx), for example, is pretty different even in layout.